scm-webapp/src/main/java/sonia/scm/plugin/PluginCenter.java

```diff
@@ -42,52 +42,69 @@
 @Singleton
 public class PluginCenter {
 
-  private static final String CACHE_NAME = "sonia.cache.plugins";
+  private static final String PLUGINS_CACHE_NAME = "sonia.cache.plugins";
+  private static final String PLUGIN_SETS_CACHE_NAME = "sonia.cache.pluginSets";
 
   private static final Logger LOG = LoggerFactory.getLogger(PluginCenter.class);
 
   private final SCMContextProvider context;
   private final ScmConfiguration configuration;
   private final PluginCenterLoader loader;
-  private final Cache<String, Set<AvailablePlugin>> cache;
+  private final Cache<String, Set<AvailablePlugin>> pluginsCache;
+  private final Cache<String, Set<PluginSet>> pluginSetsCache;
```

Comment:
I'm not sure whether it would be better to use a single Cache with a class like the one below to avoid inconsistence and redundant refetch logic in `getAvailablePlugins` and `getAvailablePluginSets`.

```java
  @AllArgsConstructor
  @Getter
  private static class CachedPlugins {
    private final Set<AvailablePlugin> plugins;
    private final Set<PluginSet> pluginSets;
  }
```